react/features/base/media/actions.js

```diff
@@ -35,10 +35,10 @@ export function muteMedia(dataJsonString: string) {
         const { kind, muted } = JSON.parse(dataJsonString);
 
         switch (kind) {
-        case 'videoinput':
-            dispatch(muteVideo(muted, 'video')); // TODO audioOnly?
+        case MEDIA_TYPE.VIDEO:
+            dispatch(muteVideo(muted, MEDIA_TYPE.VIDEO));
```

Comment:
do we need MEDIA_TYPE.VIDEO in params? the method name implies that is wont be called with audio